realtor-service/src/main/resources/application.properties

```diff
@@ -1,2 +1,2 @@
 spring.application.name=realtor-service
-feign.hystrix.enabled=true
+feign.hystrix.enabled=true
```

Comment:
It will affect next changes. In a case when we add a new property to the end of the file, we will get a diff with 2 line (but we changed only one.)